hera_mc/tests/test_rtp.py

```diff
@@ -284,7 +287,7 @@ def test_errors_rtp_process_record(self):
         self.assertTrue(len(obs_result), 1)
 
         self.assertRaises(ValueError, self.test_session.add_rtp_process_record, 'foo',
-                          *self.status_values[1:])
+                          *((self.status_values)*2))
```

Comment:
I think it'd be clearer not to re-use `status_values` for this purpose. Also, the inner parentheses aren't needed.